charts/astronomer/templates/houston/api/houston-backend-secret.yaml

```diff
@@ -17,6 +17,6 @@ data:
   # in prisma1 has hardcoded schema, so for now to migrate to prisma2 we need to hardcode it too.
   connection: {{ (printf "postgresql://%s:%s@%s:%s/%s?schema=houston%24default" .Values.houston.backendConnection.user .Values.houston.backendConnection.pass .Values.houston.backendConnection.host (.Values.houston.backendConnection.port | toString) .Values.houston.backendConnection.db) | b64enc | quote }}
   {{- else }}
-  connection: ""
+  connection: {{ randAlphaNum 5 | b64enc | quote }}
```

Comment:
@andriisoldatenko @sjmiller609 so this is just adding a little 5 char string that will get overwritten soon after creation?